Source/WebCore/html/track/VideoTrackList.h

```diff
@@ -52,6 +53,7 @@ class VideoTrackList final : public TrackListBase {
     VideoTrack* lastItem() const { return item(length() - 1); }
     VideoTrack* selectedItem() const;
     void append(Ref<VideoTrack>&&);
+    std::optional<SpatialVideoMetadata> spatialVideoMetadata() const;
```

Comment:
Let's not add `SpatialVideoMetadata` to `VideoTrackList` itself. Clients can just do `selectedItem() ? selectedItem()->spatialVideoMetadata() : nullptr`.